src/Components/People/DisplayPerson.tsx

```diff
@@ -0,0 +1,160 @@
+import {
+  CardContent,
+  CardActions,
+  Fab,
+  Typography,
+  Button,
+} from "@mui/material";
+import EditIcon from "@mui/icons-material/Edit";
+import DeleteIcon from "@mui/icons-material/Delete";
+import { FileDetails, personDisabledDetails } from "../../Interfaces/Person";
+import PeopleServices from "../../Services/People";
+import { toast } from "react-toastify";
+import { useEffect, useState } from "react";
+import KeyboardArrowUpIcon from "@mui/icons-material/KeyboardArrowUp";
+import KeyboardArrowDownIcon from "@mui/icons-material/KeyboardArrowDown";
+import { StyledLink } from "../../Styles/styledComponents";
+
+const PersonDisabledDetails = (props: personDisabledDetails) => {
+  const {
+    setAreSlotsEnabled,
+    areSlotsEnabled,
+    deletePerson,
+    id,
+    name,
+    favoriteAnimal,
+    favoriteColor,
+    favoriteFood,
+    files,
+  } = props;
+  const [personFiles, setPersonFiles] = useState<FileDetails[]>([]);
+  const [isFilesOpen, setIsFilesOpen] = useState<boolean>(false);
+
+  useEffect(() => {
+    if (files)
+      setPersonFiles(
+        files.map((file) => {
+          const nameArray = file.name.split("-");
+          const name = nameArray[1];
+          return { name: name, url: file.url };
+        })
+      );
+  }, []);
+
+  const uploadFile = async (file: File) => {
+    const formData = new FormData();
+    formData.append("file", file);
+    await PeopleServices.uploadFile(id, formData)
+      .then((res) => {
+        setPersonFiles((files) => {
+          return [...files, res.data];
+        });
+        return toast.success("The file has been uploaded!");
+      })
+      .catch((err) => toast.error(err.response.data));
+  };
+
+  return (
+    <>
+      <CardContent>
+        <Typography variant="h5" component="div"></Typography>
+        <Typography sx={{ mb: 1.5 }} color="text.secondary">
+          {name}
+        </Typography>
+        <Typography variant="body2" sx={{ mb: 1.5 }} marginLeft={2}>
+          Favorite color: {favoriteColor}
+        </Typography>
+        <Typography variant="body2" sx={{ mb: 1.5 }} marginLeft={2}>
+          Favorite food: {favoriteFood}
+        </Typography>
+        <Typography variant="body2" sx={{ mb: 1.5 }} marginLeft={2}>
+          Favorite animal: {favoriteAnimal}
+        </Typography>
```

Comment:
Why the code duplication here? Can't they all be in the same `Typography` with a `line-height` for spacing?